oe_webtools_analytics/src/Utils/WebtoolsAnalyticsIndex.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types = 1);
+
+/**
+ * Contains the json index.
+ *
+ * It will be found on the page when the analytic script tag is rendered.
+ *
+ * @see https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?spaceKey=webtools&title=Piwik
+ */
+
+namespace Drupal\oe_webtools_analytics\Utils;
+
+/**
+ * Class WebtoolsAnalyticsSearch.
+ *
+ * @package Drupal\oe_webtools_analytics\Entity
+ */
+class WebtoolsAnalyticsIndex {
```

Comment:
Why not move these properties to the event?
There are some instances where they are not used yet (jsonSerialize of the Even class)